apps/datahub-e2e/src/e2e/datasets.cy.ts

```diff
@@ -2,7 +2,6 @@ import 'cypress-real-events'
 
 describe('datasets', () => {
   beforeEach(() => {
-    cy.clearCookies()
```

Comment:
Where are the cookies cleaned now?

Reply:
cypress clears cookies between each run by default: https://docs.cypress.io/api/commands/clearcookies